test/unit/src/common/components/select-repository-list/t_select-repository-list.js

```diff
@@ -184,4 +185,22 @@ describe('<SelectRepositoryListComponent /> instance', function() {
       expect(spy).toHaveBeenCalledWith(addRepos(testProps.reposToAdd));
     });
   });
+
+  context('searching repositories', function() {
+    let wrapper;
+    let spy;
+
+    beforeEach(function() {
+      spy = spyOn(props, 'dispatch');
```

Comment:
We don't need a spy here, as we don't dispatch any actions in this test.

Same applies to unnecessary `afterEach` block that clears the spy.